addons/mail/static/src/discuss/core/web/discuss_sidebar_categories.xml

```diff
@@ -33,6 +34,7 @@
             <t t-foreach="filteredThreads(category)" t-as="thread" t-key="thread.localId" t-call="mail.DiscussSidebarChannel">
                 <t t-set="thread" t-value="thread"/>
             </t>
+            <span t-if="store.channels.status === 'fetching'" class="ms-4 o-visible-short-delay">Loading…</span>
```

Comment:
I'm not sure what is the purpose of `o-visible-short-delay` but there's still a strange flicker when opening messaging menu when it loads data fast.